pkgs/applications/misc/keepassx/community.nix

```diff
@@ -57,6 +58,11 @@ stdenv.mkDerivation rec {
   NIX_LDFLAGS = optionalString stdenv.isDarwin "-rpath ${libargon2}/lib";
 
   patches = [
+    (fetchpatch {
+      # autostart: Linux: Exec= filename not absoloute path
```

Comment:
```suggestion
      # autostart: Linux: Exec= filename not absolute path
```

Reply:
Fixed also in the commit message, thanks.